bosh_cli/lib/cli/commands/task.rb

```diff
@@ -119,16 +119,16 @@ def get_last_task_id(verbose = 1)
     def show_tasks_table(tasks)
       return if tasks.empty?
       tasks_table = table do |t|
-        t.headings = "#", "State", "Timestamp", "User", "Deployment", "Description", "Result"
+        t.headings = "#", "State", "Started", "Last Timestamp",  "User", "Deployment", "Description", "Result"
```

Comment:
"Last Timestamp" sounds a bit weird and doesn't really give context. Would "Last Update" or "Updated" or "Last Activity" perhaps be a bit more descriptive and helpful to users?